src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="http://localserver:3000/api"
+    method="post"
+    data-qa="big"
+    >
+      <input type="search"
+      data-qa="keypress"
+      placeholder="Try &quot;Los Angeles&quot;"
+      class="search-input"
+      >
+    </form>
+    <form
+    data-qa="small"
+    action="http://localserver:3000/api"
```

Comment:
set correct action from task's repository

Reply:
Haven`t found this info.Could you hint me? @alexandra-protyanova